### DISCUSSION TITLE
Ignoring devices

### DISCUSSION THREAD

**Discussion by amery78** [samemory/homebridge-eufy-security-control#253] [Q&A] [answered]
Hi all and first of all thank you for this excellent plugin.
I would like to know how to find the ID/Name to be added in order to ignore a device/station ?

Thank you for your help !

Kr,

Axel

**Comment by lenoxys** [comment]
Hello,

You can do the wizard process under settings to see the list and ignore or not device. It will display the name and the type of device.

**Reply by amery78** [reply]
Hi @lenoxys and thank you for your answer.
I can't find the wizard, I'm using hoobs and not home bridge, might be the reason...

Thx

**Reply by lenoxys** [reply]
Yeah as per our documentation HOOBs don't support custom UI so you will need to do it manually there are some example on the plug-in wiki.

**Reply by amery78** [reply]
Could you give me a link please ? Despite my research I can't find example on how do it manually...
Thank you very much !

**Reply by lenoxys** [reply]
It's on the GitHub page

https://github.com/samemory/homebridge-eufy-security/wiki/Configuration

**Reply by amery78** [reply]
Thank you, I can see how to put the parameter in Hoobs (pretty clear) but not how to grad the ID of the device I want to ignore ;)

**Comment by amery78** [comment, marked as answer]
Hello all, I finally found how to find device/station ID in hoobs
In Homebridge-Eufy-Security plugin select debug mode for logs.
Go into Log windows in Hoobs and click on Debug Log button...
You will find here all needed information...
Thank Lenoxys for your support :)

Axel
